### DISCUSSION TITLE
Hello fam welcome to the community

### DISCUSSION THREAD

**Discussion by Teamsupo** [community/community#123641] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello fam welcome to the community

**Comment by Teamsupo** [comment]
Hello fam welcome to the community

**Comment by ebndev** [comment]
Hi @Teamsupo, thanks for being a part of the GitHub Community!

Please feel free to say hello in our New to GitHub [Introduction Thread](https://github.com/orgs/community/discussions/23204) 😄
